frontend/src/components/Calendar/index.tsx

```diff
@@ -50,24 +55,42 @@ function Calendar({ channelId, handleCloseCalendar }: Props) {
       </Styled.Weekdays>
 
       <Styled.Days>
-        {getCurrentDays().map((day, index) => (
-          <Link
-            key={index}
-            to={`/feed/${channelId}/${ISOConverter(
-              `${date.getFullYear()}-${MONTHS[date.getMonth()]}-${day}`
-            )}`}
-          >
+        {getCurrentDays().map((day, index) => {
+          const isBlank = day === "";
+          const isFuture = day > todayDate.getDate() && isFutureMonth();
+          const isCurrentDay = day === todayDate.getDate() && isCurrentMonth();
+
```

Comment:
getDate 메서드가 반복해서 사용되는데, 앞에다 getDate, getFullYear, getMonth 를 변수에 담아주고 사용하는건 어떨까요 ~? 👀 급한건 아니니 리팩터링할때 같이 이야기 나누어보아용

```suggestion
        {getCurrentDays().map((day, index) => {
        
         const  date = todayDate.getDate();
         const month = todayDate.getMonth();
         const year = todayDate.getFullYear();
         
          const isBlank = day === "";
          const isFuture = day > date&& isFutureMonth();
          const isCurrentDay = day === date && isCurrentMonth();

```

Reply:
넵 이 부분은 추가로 같이 이야기 해보아용!